components/NavigationDrawer/src/private/MDCBottomDrawerContainerViewController.m

```diff
@@ -643,6 +643,21 @@ - (void)updateViewWithContentOffset:(CGPoint)contentOffset {
   CGFloat fullscreenHeaderHeight =
       self.contentReachesFullscreen ? self.topHeaderHeight : [self contentHeaderHeight];
 
+  CGFloat contentHeight = self.contentViewController.preferredContentSize.height +
```

Comment:
calling the preferredContentSize of the contentViewController every contentOffset change of the scroll will cause degraded performance in clients with a calculated (more complex) preferredContentSize for their view controller. This was seen with OneGoogle, hence the addition of _contentVCPreferredContentSizeHeightCached

Reply:
The same could possibly be said for the headerViewController preferredContentSize.

Reply:
Addressed in [#04c8695](https://github.com/material-components/material-components-ios/pull/8484/commits/04c869573f5d35f2f0f18ae0aed189d0ad28f327)